### PR TITLE
first pass at ingestion docs

### PR DESCRIPTION
**Pull request by psFried**
Documentation of the behavior in: https://github.com/estuary/flow/pull/60

### CODE REVIEW

File: concepts.rst [hunk 1]
```diff
@@ -277,6 +277,12 @@ collapsing a torrent of ingested documents into a trickle.
     transaction. This natural back-pressure affords *plenty* of
     opportunity for data reductions, while minimizing latency.
 
+    In other words, as a target database becomes busier or slower,
```
Comment (jgraettinger) [inline concepts.rst]: bad merge ? I had moved this section

File: docs/ingest.rst [hunk 2]
```diff
@@ -0,0 +1,239 @@
+
+Ingesting Data
+==============
+
+There are a number of different options for how to ingest data into Flow:
+
+* `HTTP PUT or POST requests`
+* `Stream data over a Websocket` in either `CSV, TSV`, or `JSON` formats
+* Write data using a `Gazette Consumer <https://gazette.readthedocs.io/en/latest/consumers-concepts.html>`_ application
```
Comment (jgraettinger) [inline docs/ingest.rst]: I don't think we want to support direct writes from consumers other than Flow; just drop this bullet?

**Review by jgraettinger — APPROVED** [review]
LGTM % comments !

**Review by dyaffe — COMMENTED** [review]
LGTM as well!

File: concepts.rst (hunk 1 above)
Comment (psFried) [reply concepts.rst]: Good catch. Yeah, for some reason git had trouble with this section.

**Pull request merged**